libnavui-voice/src/main/java/com/mapbox/navigation/ui/voice/api/MapboxVoiceInstructionsPlayer.kt

```diff
@@ -96,6 +96,7 @@ class MapboxVoiceInstructionsPlayer @JvmOverloads constructor(
         clean()
         filePlayer.clear()
         textPlayer.clear()
+        audioFocusDelegate.abandonFocus()
```

Comment:
I suggest to move the call to `clean` function. 
Also we should test that `abandonFocus` is called.

Reply:
Implemented both